txcache/selection.go

```diff
@@ -8,7 +8,8 @@ import (
 func (cache *TxCache) doSelectTransactions(session SelectionSession, gasRequested uint64, maxNum int, selectionLoopMaximumDuration time.Duration) (bunchOfTransactions, uint64) {
 	bunches := cache.acquireBunchesOfTransactions()
 
-	return selectTransactionsFromBunches(session, bunches, gasRequested, maxNum, selectionLoopMaximumDuration)
+	return selectTransactionsFromBunches(session, bunches, gasRequested,
```

Comment:
Oh, I see, since `selectTransactionsFromBunches` is not a member function of `TxCache` - at least, not yet :+1: